OpenTrons/opentrons-api
# fix(app): Various quick transfer fixes

Fix RQA-2908, RQA-2848, RQA-2906


# Overview
This PR addresses a number of quick transfer bugs and loose ends - see change log


## Test Plan and Hands on Testing
1. Check that the Quick Transfer FF no longer exists
2. Going through the QT flow, see that if you select a 96 well plate, you see one of the below options 
3. Check that if Mix is enabled, you can enter the setting and snackbar won't appear
4. Enable mix, set values then go back and disable, see that setting is closed and previous values wiped
5. For a multi-dispense, see that on the first screen of the pipette path option you only see "Multi-dispense" text, and on the summary page you also see the volume and blowout location

<img width="1016" alt="Screen Shot 2024-08-06 at 3 24 38 PM" src="https://github.com/user-attachments/assets/11a84203-5c7e-4d16-8255-02626ba5c0a0">
<img width="1009" alt="Screen Shot 2024-08-06 at 3 24 28 PM" src="https://github.com/user-attachments/assets/d4b81362-1435-4dcd-9d7b-297344ffde3c">




## Changelog
1. Remove feature flag for quick transfer
6. Fix Mix setting disabled logic so that the snackbar appears only when the setting is disabled
7. Add missing `onBack` call after saving a Mix setting as disabled
8. Fix multi-dispense text display within pipette path setting
9. Swap out 96-well plate definitions with a circular or square well def that has a large touch target for source and destination well selection



## Review requests
Look through code, this has been tested according to the plan above


## Risk assessment
Low
